program/squeezedet/image_benchmark.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+export PYTHONPATH="$PYTHONPATH:$CK_ENV_DEMO_SQUEEZEDET_SRC"
+export TF_CPP_MIN_LOG_LEVEL=3 # supress TF debug output
+
+CHECKPOINT=$CK_ENV_MODEL_SQUEEZEDET_MODEL
+NET=$CK_ENV_MODEL_SQUEEZEDET_ID
+INPUT=$CK_ENV_DEMO_SQUEEZEDET_ROOT/data/sample.png
```

Comment:
Should take images from a dataset?

Reply:
I've just reused approach of `default` command here. Taking a dataset path we could process 'real' batches with different images. But there are several questions:
- Which tags should we use for search for appropriate dataset? 
- What should we do if specified dataset does not contain `batch_size * batch_count` images?

Reply:
I've done it as in 'use_continuous'